test/screens/v2/widget_instance/departures_test.exs

```diff
@@ -950,11 +966,171 @@ defmodule Screens.V2.WidgetInstance.DeparturesTest do
         header: %Header{title: "Section Header", read_as: "A special audio-only value"}
       }
 
+      screen = struct(Screen, %{app_id: :gl_eink_v2})
+
+      now = ~U[2020-01-01T00:00:00Z]
+
       assert %{
                type: :normal_section,
                departure_groups: [],
                header: "A special audio-only value"
-             } = Departures.audio_serialize_section(section, nil)
+             } =
+               Departures.audio_serialize_section(
+                 section,
+                 screen,
+                 now
+               )
+    end
+
+    test "busway_v2 return 1 departure time if first departure is > 2 minutes away" do
+      now = ~U[2020-01-01T00:00:00Z]
+
+      section = %{
+        type: :normal_section,
+        rows: [
+          %Departure{
+            prediction: %Prediction{
+              arrival_time: ~U[2020-01-01T00:05:00Z],
+              route: %Route{type: :subway},
+              trip: %Trip{headsign: "Test"},
+              stop: %Stop{id: "place-test"}
+            }
+          },
+          %Departure{
+            prediction: %Prediction{
+              arrival_time: ~U[2020-01-01T02:01:00Z],
+              route: %Route{type: :subway},
+              trip: %Trip{headsign: "Test"},
+              stop: %Stop{id: "place-test"}
+            }
+          }
+        ],
+        header: %Header{title: "Section Header"}
+      }
+
+      screen = struct(Screen, %{app_id: :busway_v2})
```

Comment:
Any particular reason you used `struct` for these? It seems equivalent to `%Screen{app_id: :busway_v2}`, but with less compile-time safety.

Reply:
Just to automatically apply the enforced keys that we don't care about. Creating the `struct` without some hardware specific fields throws a compilation error.